pkg/agent/datamodel/versions.go

```diff
@@ -502,13 +505,13 @@ func IsKubernetesVersionGe(actualVersion, version string) bool {
 GetLatestPatchVersion gets the most recent patch version from a list of semver versions
 given a major.minor string.
 */
-func GetLatestPatchVersion(majorMinor string, versionsList []string) (version string) {
+func GetLatestPatchVersion(majorMinor string, versionsList []string) string {
 	// Try to get latest version matching the release.
-	version = ""
+	version := ""
```

Comment:
nit: 
```suggestion
	var version string
```

Reply:
I want to say the var version string threw a linter, but I'll double check on that

Reply:
I was wrong! whoops